xbmc/guilib/guiinfo/PlayerGUIInfo.cpp

```diff
@@ -331,6 +331,14 @@ bool CPlayerGUIInfo::GetLabel(std::string& value, const CFileItem *item, int con
     case PLAYER_PROCESS_VIDEOHEIGHT:
       value = StringUtils::FormatNumber(CServiceBroker::GetDataCacheCore().GetVideoHeight());
       return true;
+    case PLAYER_PROCESS_VIDEOBITRATE:
```

Comment:
Wouldn't it make more sense to expose this as an integer infolabel? Also what does happen if it's not possible to extract the bitrate? Looking at the code it seems you'll have 0 bps exposed on the interface.
Maybe if you make this an integer infolabel you can then set some condition like Integer.IsGreater to avoid showing it on the skin

Reply:
if the value is not set the value is kept empty and in the skin nothing will be displayed,
if you handle it as number you will lose the number formatting and make more hard understand the big number in the GUI

Reply:
Right, sorry for the noise